src/workerd/server/server.c++

```diff
@@ -1156,6 +1156,13 @@ public:
 
   class ActorNamespace {
   public:
+    struct LocalActor {
+      kj::Own<Worker::Actor::Impl> actor;
+
+      struct None{};
+      kj::OneOf<None, Worker::Actor*, kj::Promise<void>> hibernationTaskOrActor = None{};
```

Comment:
It appears that we never set `hibernationTaskOrActor` to a promise in this PR. Likewise, I don't think we ever actually set it to `None{}` outside of this default ctor. Perhaps `kj::Maybe<Actor*>` for now?